templates/chartmuseum/chartmuseum-cm.yaml

```diff
@@ -11,6 +11,9 @@ data:
   CACHE_REDIS_ADDR: "{{ template "harbor.redis.host" . }}:{{ template "harbor.redis.port" . }}"
   CACHE_REDIS_DB: "{{ template "harbor.redis.chartmuseumDatabaseIndex" . }}"
   BASIC_AUTH_USER: "chart_controller"
+{{- if eq .Values.chartmuseum.absoluteUrl }}
```

Comment:
I think the `eq` should be removed as it needs two parameters.

Reply:
Oops!

Reply:
@ywk253100 fixed in f7bcd80b34c5623555efbd2ec82e6cbe8430129b